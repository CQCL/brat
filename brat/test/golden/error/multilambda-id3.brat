f(Bool) -> { Qubit -o Qubit }
f = { true => |;||false =>|;|;|}
