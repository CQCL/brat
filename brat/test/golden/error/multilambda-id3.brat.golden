Error in test/golden/error/multilambda-id3.brat@FC {start = Pos {line = 2, col = 15}, end = Pos {line = 2, col = 18}}:
f = { true => |;||false =>|;|;|}
              ^^^

  Elab error "Noun required at this position"

